### DISCUSSION TITLE
How to set the output folder to <directory>?

### DISCUSSION THREAD

**Discussion by dertuxmalwieder** [enzo1982/freac#498] [Help & Support] [answered]
I'm using fre:ac 1.1.7 on macOS. What I usually want to do is convert a folder of files into a different format *without changing the output folder*. fre:ac's documentation suggests that I could just set the output folder to `<directory>`; however, the field for the output folder is read-only and all I can do is "open" and "choose".

![Screenshot](https://imgur.com/yxFXket.png)

How can I set `<directory>` there? It is *annoying* to click through the hard disk structure every time. :-)

**Comment by enzo1982** [comment]
The placeholders like `<directory>` and `<filename>` are only valid for the filename pattern field.

There are two options to achieve what you intend to do:

1. Enable the _Use input file folder if possible_ option on the _Output files_ page in the configuration dialog. Then set the filename pattern to e.g. `<filename>`.
2. Set the output folder to `/` and the filename pattern to e.g. `<directory>/<filename>`.

**Reply by dertuxmalwieder** [reply]
I enabled the option mentioned in 1., but it still defaults to whatever I had used previously. Is the filename pattern relevant?

**Reply by enzo1982** [reply, marked as answer]
The _Output folder_ field will always show the configured output folder even with the _Use input file folder if possible_ option enabled. When the option is enabled, the configured output folder will be use as a fallback, e.g. when ripping CDs.

The filename pattern is still relevant when the option is enabled. Let's say you have a file `/Volumes/Music/Album1/File1.flac `and are converting that to MP3. If your output folder is `/Users/tux0r/Downloads` and the output filename pattern is `<filename>`, it will create the file `/Users/tux0r/Downloads/File1.mp3`.

When you enable the _Use input file folder if possible_ option, it should create the file `/Volumes/Music/Album1/File1.mp3`.

**Reply by dertuxmalwieder** [reply]
Got it! It works now as intended, even with a different filename pattern. Thank you a ton!
